velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -494,22 +500,28 @@ core::PlanNodePtr SubstraitVeloxPlanConverter::toVeloxPlan(
 core::PlanNodePtr SubstraitVeloxPlanConverter::toVeloxPlan(
     const ::substrait::Plan& substraitPlan,
     memory::MemoryPool* pool) {
-  // Construct the function map based on the Substrait representation.
-  constructFunctionMap(substraitPlan);
+  if (!checkTypeExtension(substraitPlan)) {
```

Comment:
Let's make this a little shorter:

```
VELOX_CHECK(checkTypeExtension(substraitPlan), ....);
```

(and remove if-else.)

Reply:
Done.